cds/modules/records/tasks.py

```diff
@@ -223,3 +228,108 @@ def file_integrity_report():
         recipients = [current_app.config['CDS_ADMIN_EMAIL']]
         msg = Message(subject, sender=sender, recipients=recipients, body=body)
         current_app.extensions['mail'].send(msg)
+
+
+def format_subformats_integrity_report(report):
+    """Format the email body for the file integrity report."""
+    lines = []
+
+    for entry in report:
+        lines.append('Message: {}'.format(str(entry.get('message'))))
+        lines.append('Record: {}'.format(entry.get('recid')))
```

Comment:
Can you please add the link too as done [here](https://github.com/CERNDocumentServer/cds-videos/blob/cdslabs/cds/modules/records/tasks.py#L170)?
Something like:
```
Record: 1 - https://videos.cern.ch/record/1
```